q2_vizard/plugin_setup.py

```diff
@@ -54,7 +55,41 @@
                      ' plotted on the x-axis.',
         'y_measure': 'Numeric measure from the input Metadata that should be'
                      ' plotted on the y-axis.',
-        'color_by_group': 'Categorical measure from the input Metadata that'
-                          ' should be used for color-coding the scatterplot.',
+        'color_by': 'Categorical measure from the input Metadata that'
+                    ' should be used for color-coding the scatterplot.',
         'title': 'The title of the scatterplot.'}
 )
+
+
+plugin.visualizers.register_function(
+    function=lineplot,
+    inputs={},
+    parameters={
+        'metadata': Metadata,
+        'x_measure': Str,
+        'y_measure': Str,
+        'replicates': Bool,
+        'average': Str % Choices('median', 'mean'),
+        'facet_by': Str,
```

Comment:
Should this also be `color_by` to match the other?

Reply:
okay so this is a good question - because technically, yes, this is assigning the colors. but this is also what's actually creating the faceting, which is more than just color assignment (like in scatterplot). honestly i'm open to either param name - it's a little in between the two, because boxplot has a facet_by param that's doing the grouping (but that's a bit more distinct than this in terms of the groupings).

Reply:
I think it may be useful to presume a user would expect a facet to involve subplots (which in the Vega language is pretty much no different from a group mark). So I would err on the side of `group_by` or `color_by`, since the only reasonable way to have multiple colored lines is to have grouped the data, so I think either implies about the same thing. But it would be reasonable to think that a `facet_by` means that each line gets its own subplot.

Reply:
fair enough! i like `group_by` - this feels more distinct than just the coloring. will update that now, and then this should be ready for another review!